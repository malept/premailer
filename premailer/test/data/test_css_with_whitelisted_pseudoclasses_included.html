<html>
<head>
<style type="text/css">
a { color:red; }
a,a:hover,
a:visited { border:1px solid green; }
p::first-letter {float: left; font-size: 300%}
p:first-child {color: blue}
</style>
</head>
<body>
<p>First Paragraph</p>
<p>Second Paragraph</p>
<a href="#" style="color:red;border:1px solid green">Page</a>
</body>
</html>
