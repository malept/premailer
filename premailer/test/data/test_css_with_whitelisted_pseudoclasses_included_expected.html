<html>
<head>
<style type="text/css">a:hover{border:1px solid green}
a:visited{border:1px solid green}
p::first-letter{float:left;font-size:300%}
</style>
</head>
<body>
<p style="color:blue">First Paragraph</p>
<p>Second Paragraph</p>
<a href="#" style="color:red;border:1px solid green">Page</a>
</body>
</html>
